test/clients/proposal_test.rb

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "../test_helper.rb"
+require "y2firewall/clients/proposal"
+
+describe Y2Firewall::Clients::Proposal do
+  let(:client) { described_class.new }
+
+  describe "#initialize" do
+    it "instantiates a new proposal settings" do
```

Comment:
well, I think it is not needed to test that it is instantiated, as it is common ruby functionality.

Reply:
well, it is part of the code and it is public, it is a simple test but even so..

Reply:
BTW, this is really a test for lib/y2firewall/clients/proposal_test.rb so will move all the lib tests to a folder that really match its path